### DISCUSSION TITLE
Nomination of @lsiepel

### DISCUSSION THREAD

**Discussion by kaikreuzer** [openhab/openhab-addons#16184] [Add-on Maintainers Discussions]
Dear @openhab/add-ons-maintainers,

I'd like to nominate @lsiepel for becoming a maintainer of this repo.

I've watched his growing engagement throughout the year 2023 with joy. He started with some simple contributions like code clean-ups and learned about the openHAB code over time and is by now one of the most active contributors and also code reviewers. I think his [long list of PRs](https://github.com/openhab/openhab-addons/pulls?q=author%3Alsiepel) as well as his [various review comments](https://github.com/openhab/openhab-addons/pulls?q=is%3Apr+commenter%3Alsiepel) show his dedication.

Please vote with 👍 or 👎 here on this post. Thanks!

**Comment by kaikreuzer** [comment]
Awesome, thanks for the votes! The voting period is over and as we do not have any vetos by a maintainer, I am very happy to welcome @lsiepel in the round of @openhab/add-ons-maintainers! 🎉

**Reply by lsiepel** [reply]
Thanks, happy to help where possible!
